lib/versioned/^3.7.0/index.js

```diff
@@ -38,7 +41,8 @@ function execute(opts, env) {
     }
     if (opts.tasks) {
       var tree = taskTree(gulpInst.tasks);
-      tree.label = 'Tasks for ' + chalk.magenta(tildify(env.configPath));
+      tree.label = isString(config.description) ? config.description :
```

Comment:
It's not enough to check `isString`.  We don't want to print an empty string description